tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -2324,6 +2325,106 @@ def test_auto_dtype(self, use_py_transceiver, mocker):
+
+    MODEL_NAME = "nvidia/Qwen3.5-397B-A17B-NVFP4-V2"
+    MODEL_PATH = f"{llm_models_root()}/Qwen3.5-397B-A17B-NVFP4-V2"
+    EXTRA_EVALUATOR_KWARGS = {
+        GSM8K: {
+            "apply_chat_template": True,
+            "fewshot_as_multiturn": True,
+            "chat_template_kwargs": {
+                "enable_thinking": False,
+            },
+        },
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)